src/functions/guild/editEmbed.js

```diff
@@ -0,0 +1,58 @@
+module.exports = async d => {
+  const data = d.util.aoiFunc(d);
+  var [channelID = d.channel?.id, messageID = d.message?.id, index = 1, option = 'description', content] = data.inside.splits;
+ index = index - 1;
+const channel = await d.util.getChannel(d, channelID);
+if (!channel) return d.aoiError.fnError(d, 'channel', {}, {inside: data.inside});
+const message = await d.util.getMessage(channel, messageID);
+if (!message) return d.aoiError.fnError(d, 'message', {}, {inside: data.inside});
+function isNumber(value) {
+  return typeof value === "number" && !Number.isNaN(value);
+}
```

Comment:
just use isNaN(Number), aoijs uses the concept of strings therefore numbers dont exist unless converted into using Number() or similar iirc

Reply:
Okay